tool/src/org/antlr/v4/codegen/target/GoTarget.java

```diff
@@ -59,42 +56,6 @@ protected Set<String> getReservedWords() {
 		return reservedWords;
 	}
 
-	@Override
-	protected void genFile(Grammar g, ST outputFileST, String fileName) {
-		super.genFile(g, outputFileST, fileName);
-		if (DO_GOFMT && !fileName.startsWith(".") /* criterion taken from gofmt */ && fileName.endsWith(".go")) {
-			gofmt(new File(getCodeGenerator().tool.getOutputDirectory(g.fileName), fileName));
-		}
-	}
-
-	private void gofmt(File fileName) {
```

Comment:
@jimidle do we want to run go fmt or should this be removed?

Reply:
It looks strange to me that ANTLR core depends on `gofmt` tool and I don't clearly understand why this formatting is required at all.